### PR TITLE
Simplify FAQ Code

### PR DESCRIPTION
**Pull request by lkiesow**
This patch simplifies the FAQ code by removing as much custom HTML as
possible, using default Markdown and CSS instead.

In addition, this is a review of the FAQ content which I updated when
going through the HTML code.

### CODE REVIEW

**Review by tibroc — APPROVED** [review]
lgtm

File: faq.md [hunk 1]
```diff
@@ -1,111 +1,67 @@
...
+Several additional data products are available any may find their way into our API in the future.
+A few examples include:
 
-<details>
-<summary><h3> > How to get in contact with emissions API?</h3></summary>
-  <div style="margin-left: 20px;">
-    Via <a href="mailto:info@emissions-api.org">info@emissions-api.org</a>, <a href="https://twitter.com/emissions_api">twitter</a>, <a href="https://mastodon.social/@emissions_api">mastodon</a> or by leaving a comment e.g. on an issue.<br>
-  </div>
- <br></details>
+- Ozone (O³)
```
Comment (tibroc) [inline faq.md]: These are actually all subscript instead of superscript.

**Review by tibroc — CHANGES_REQUESTED** [review]
Please just change the superscript of the chemical formulas to subscript.

File: faq.md (hunk 1 above)
Comment (lkiesow) [reply faq.md]: Makes sense. Fixed.

**Comment by TylerBird** [comment]
yeah, whatever lgtm. 
If you like this way more, it is fine by me.

**Pull request merged**